risc0/zkvm/methods/build.rs

```diff
@@ -34,5 +34,10 @@ fn main() {
         ),
     ]);
 
-    embed_methods_with_options(map);
+    if cfg!(feature = "test-exact-cycles") {
+        std::env::set_current_dir("../../").unwrap();
```

Comment:
This part worries me. This is probably fine for now, but in the future I think we need to be able to configure the root of the docker context.

Reply:
100% agree! Tracking issue here: https://github.com/risc0/risc0/issues/913